dev.env

```diff
@@ -22,3 +22,4 @@ REDIS_URL=
 CMC_API=https://pro-api.coinmarketcap.com
 CMC_API_KEY=
 NEXT_PUBLIC_UNREADY_COMPONENTS=a
+NEXT_PUBLIC_LINK_DAPP=a
```

Comment:
Curious, why is it set to `a` rather than being empty?

Reply:
No particular reason, it can be empty or not. It just enables the newer layout.